### PR TITLE
LVNOD-80-Integrated guests list into event details

### PR DESCRIPTION
**Pull request by AlbaMontis**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-80)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-88)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-90)

## Code reviewers

- [ ] @yarvanya
- [x] @denisbodnar
- [x] @shyyka

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

### CODE REVIEW

**Review by denisbodnar — DISMISSED** [review]

**Review by yarvanya — DISMISSED** [review]

File: app/src/containers/Event/EventDetails.js [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+  }
+
+  deleteGuestEmail = i => () => {
+    const guest = this.props.guests[i];
+    this.props.guestActions.deleteGuest(guest);
   }
 
   render() {
 
     const id = this.props.params.id;
```
Comment (4asof4ik) [inline app/src/containers/Event/EventDetails.js]: {id, ......} = this.porps

**Review by shyyka — DISMISSED** [review]

File: app/src/containers/Event/EventDetails.js [hunk 2]
```diff
@@ -37,7 +54,23 @@ class EventDetails extends React.Component {
               { this.props.children }
             </Tab>
             <Tab eventKey={2} title='Guests'>
-              <Link className='list-group-item' to={'/events/' + id + '/guests'}>Guests</Link>
+              <ListGroup>
+                <ListGroupItem className='clearfix'>
+                  <h4 className='pull-left'>Guests list</h4>
+                </ListGroupItem>
+                {guests.length ? guests.map((guest, index) =>
```
Comment (4asof4ik) [inline app/src/containers/Event/EventDetails.js]: create function guestsList and use && and ||

File: app/src/redux/invite.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        res &&
+        dispatch({
+          type: SEND_INVITES
+        });
+      });
+  };
+}
+
+export function getEmails() {
+  const eventId = location.pathname.split('/')[2];
```
Comment (4asof4ik) [inline app/src/redux/invite.js]: drop and add arg with event id

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

File: app/src/redux/invite.js (hunk 3 above)
Comment (AlbaMontis) [reply app/src/redux/invite.js]: Done.

File: app/src/containers/Event/EventDetails.js (hunk 1 above)
Comment (AlbaMontis) [reply app/src/containers/Event/EventDetails.js]: Done.

**Pull request closed without merging**